app/services/hyrax/admin_set_create_service.rb

```diff
@@ -84,7 +98,8 @@ def admin_group_name
     ##
     # @return [PermissionTemplate]
     def create_permission_template
-      permission_template = PermissionTemplate.create!(source_id: admin_set.id, access_grants_attributes: access_grants_attributes)
+      permission_template = PermissionTemplate.create!(source_id: admin_set.id,
+                                                       access_grants_attributes: access_grants_attributes.uniq)
```

Comment:
There will be dups in the grants if one of the managers from the admin set collection type is the creating user.  Adding `.uniq` to only add a user once.